test/session.js

```diff
@@ -1269,14 +1532,14 @@ describe('session()', function(){
       .get('/')
       .expect(shouldNotHaveHeader('Set-Cookie'))
       .expect(200, function(err, res){
-        if (err) return done(err);
```

Comment:
Yeah, that doesn't surprise me too much as it seemed like someone changed the test code style (e.g. to be semi-colon-less) in the time since my initial PR submission. I'll try to clean them up when I make my updates.